gapic/src/main/com/google/gapid/perfetto/views/SliceSelectionView.java

```diff
@@ -104,6 +104,10 @@ public SliceSelectionView(Composite parent, State state, SliceTrack.Slice slice)
         }
       }
 
+      if (renderStageInfo.submissionId != 0) {
+        propsBuilder.put("VkSubmissionId:", Long.toString(renderStageInfo.submissionId));
```

Comment:
Should be just "Submission ID".  There is no VkSubmissionId in the vulkan API.

Reply:
Done in a new commit.